### PR TITLE
Add Asciidoc attributes for OS and tool versions, and update them automatically

### PR DESCRIPTION
**Pull request by bgilbert**
Hardcoded versions are perpetually going out of date.  Add site-level attributes and use them instead of hardcoding.

Add script and timer workflow for updating version attributes.  Push directly to the repo, bypassing pull requests, since this is a routine metadata update.  Because we're using the repository's token, this bypasses CI on the main branch due to GitHub recursion prevention, but that shouldn't affect anything.

### CODE REVIEW

File: modules/ROOT/pages/tutorial-setup.adoc [hunk 1]
```diff
@@ -114,9 +114,9 @@ You are now ready to proceed with the xref:tutorial-autologin.adoc[first tutoria
 
 If none of the previous solutions work for you, you can still manually download Fedora CoreOS from https://getfedora.org/en/coreos/download?tab=metal_virtualized&stream=stable[getfedora.org] with:
 
-[source,bash]
+[source,bash,subs="attributes"]
 ----
-RELEASE="32.20200715.3.0" # Update this with the latest release from https://getfedora.org/en/coreos
+RELEASE="{stable-version}" # Update this with the latest release from https://getfedora.org/en/coreos
```
Comment (jlebon) [inline modules/ROOT/pages/tutorial-setup.adoc:119]: Can drop that comment now, right?

**Review by jlebon — APPROVED** [review]
Wow, this is pretty rad. Great idea!

**Review by cgwalters — APPROVED** [review]
Super cool!

**Comment by jlebon** [comment]
Looks like this is tripping on branch protection:

```
remote: error: GH006: Protected branch update failed for refs/heads/main.        
remote: error: Required status check "Validate Butane configs" is expected. At least 1 approving review is required by reviewers with write access.        
```

More info in https://github.community/t/16536.

Probably simplest is to switch to coreosbot-releng + PRs?

**Comment by travier** [comment]
This is not working (anymore?) on the official Fedora built & hosted docs: 

I've tried the same trick for the Silverblue docs and it's broken there too: https://github.com/fedora-silverblue/silverblue-docs/pull/125

**Comment by travier** [comment]
I'm testing https://github.com/fedora-silverblue/silverblue-docs/pull/126 on the Kinoite docs.

**Comment by travier** [comment]
Fix in https://github.com/coreos/fedora-coreos-docs/pull/378

**Pull request merged**